vip_hci/preproc/recentering.py

```diff
@@ -541,6 +551,10 @@ def cube_recenter_satspots(array, xy, subi_size=19, sigfactor=6, plot=True,
     lbda: 1d array or list, opt
         Wavelength vector. If provided, the subimages will be scaled accordingly
         to follow the motion of the satellite spots.
+    filter_freq: tuple of 2 elements. 
```

Comment:
Indent and use similar description as suggested above (replace the occurrences of "image" into "images" though)